salesforce/management/commands/update_partners.py

```diff
@@ -326,6 +329,10 @@ def handle(self, *args, **options):
                     )
                     created_partners = created_partners + 1
 
+            # remove partners that have been deleted from Salesforce
+            stale_partners = Partner.objects.exclude(salesforce_id__in=partner_ids)
```

Comment:
equality of array raised my eyebrow, but I trust you know what you're doing.

Reply:
:) yeah, the `__in` is checking if the salesforce id is in the array